server/migrations/20230726210735-create-skills.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Skills', {
+      skill_id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      skill_name: {
```

Comment:
these can just be `id` and `name`, `skill_*` on a table called `skills` is redundant

Reply:
but we are using them as foreign keys, wouldn't that be confusing if we get another column with id?